interface/patient_file/summary/record_disclosure.php

```diff
@@ -78,84 +78,86 @@ function submitform() {
-<div style='float: left; margin-right: 5px'><?php
-if ($editlid) {
-    ?><!--Edit the disclosures-->
-    <span class="title"><?php echo xlt('Edit Disclosure'); ?></span><?php
-} else { ?>
-        <span class="title"><?php echo xlt('Record Disclosure'); ?></span><?php
-} ?>
-</div>
+<body>
+    <div class="container" id="record-disclosure">
+        <div class="row">
+            <div class="col-12">
+                <?php
+                if ($editlid) {
+                    ?><!--Edit the disclosures-->
+                    <h2 class="title"><?php echo xlt('Edit Disclosure'); ?></h2><?php
+                } else { ?>
+                    <span class="title"><?php echo xlt('Record Disclosure'); ?></span><?php
+                } ?>
+            </div>
+            <div class="col-12">
+                <form name="disclosure_form" id="disclosure_form" method="POST" action="disclosure_full.php">
+                    <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+                    <div class="btn-group">
+                        <button class='btn btn-primary btn-save' name='form_save' id='form_save'>
+                            <?php echo xlt('Save'); ?>
+                        </button>
+                        <button class="btn btn-secondary btn-cancel" id='cancel' onclick='top.restoreSession();dlgclose()'>
+                            <?php echo xlt('Cancel'); ?>
+                        </button>
+                    </div>
 
-<form name="disclosure_form" id="disclosure_form" method="POST" action="disclosure_full.php">
-    <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+                    <input type='hidden' name='mode' value="disclosure" />
+
+                    <div class="form-group mt-3">
+                        <label><?php echo xlt('Date'); ?>:</label>
+                        <?php
+                        if ($editlid) {
+                            $dres = sqlQuery("select date,recipient,description,event from extended_log where id=?", array($editlid));
+                            $description = $dres["description"];
+                            $app_event = $dres["event"];
+                            $disc_date = $dres["date"];
+                            $recipient_name = $dres["recipient"];
+                            ?>
+                            <input type="hidden" name="disclosure_id" value="<?php echo attr($editlid); ?>">
+                            <input type="hidden" name="updatemode" value="disclosure_update">
+                            <input type='entry' size='20' class='datepicker form-control' name='dates' id='dates' value='<?php echo attr($disc_date);?>'/>&nbsp; <?php
+                        } else {
+                            ?> <input type='entry' size='20' class='datepicker form-control' name='dates' id='dates' value=''/>&nbsp;<?php
```

Comment:
Hi @bradymiller  and @tywrenn 

I guess the `type="entry"` is invalid code syntax. What do I replace?

Reply:
There are so many of these in the codebase that this will need to be a separate PR.